client/package.json

```diff
@@ -12,6 +12,8 @@
     "lint:prettier": "./node_modules/.bin/prettier \"app/**/*.{js,jsx}\" --write --single-quote  --trailing-comma all",
     "lint": "yarn lint:flow && yarn lint:prettier && yarn lint:eslint --fix",
     "storybook": "yarn build:i18n && start-storybook -p 6006",
+
+    "storybook:circleci":"build-storybook -c .storybook -o ../public/design",
```

Comment:
In addition to doing this, we'll want to put `/public/design` in `.gitignore` so that we don't commit the output to master. We only need these files when we're deploying to production.